utils/clients/django_database_client.py

```diff
@@ -0,0 +1,66 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Creates a database session for the reduction database
+"""
+import logging
+
+from model.database import DjangoORM
+from utils.clients.connection_exception import ConnectionException
+from utils.project.static_content import LOG_FORMAT
+from utils.project.structure import get_log_file
+
+logging.basicConfig(filename=get_log_file('django_database_client.log'), level=logging.INFO,
+                    format=LOG_FORMAT)
+
+
+class DatabaseClient:
+    """
+    Single access point for the mysql database
+    """
+
+    data_model = None
+    variable_model = None
```

Comment:
NIT: Would it be worth moving data_model and  variable_model inside an __init__ method with a private prefix and adding parameter descriptions?
```
class DatabaseClient:
    """
    Single access point for the mysql database
    """
    def __init__(self, data_model, variable_model):
        """
        :param data_model: 
        :param variable_model: 
        """
        self._data_model = data_model
        self._variable_model = variable_model
```

Or just adding comments to describe both variables?

Reply:
I think just adding comments in this case is sufficient. We don't actually want them to be private. Whilst I'll add some common access functions, complex queries will just be using the models directly. Either way I shall ad some comments to them